packages/grpc-native-core/index.js

```diff
@@ -122,6 +120,7 @@ var loadObject = exports.loadObject;
  * @return {Object<string, *>} The resulting gRPC object
  */
 exports.load = util.deprecate(function load(filename, format, options) {
+  const ProtoBuf = require('protobufjs');
```

Comment:
It probably doesn't matter for a deprecated function, but if it's called more than once, then doing something similar to [this](https://github.com/nodejs/node/blob/08c14d98cdb029c72549a6709c2fd0befda5a463/lib/internal/dgram.js#L27-L29) might be even more performant (if just slightly). Even though `require()` has a cache, there is still some additional overhead to using it.

Reply:
I think it's fine to keep it the way I wrote it, not only because `load` is deprecated, but also because it's expected to be called very infrequently, and it shouldn't generally be on the hot path.